src/Symfony/Component/PropertyInfo/Tests/Extractors/PhpDocExtractorTest.php

```diff
@@ -176,3 +181,13 @@ class EmptyDocBlock
 {
     public $foo;
 }
+
+class OmittedParamTagTypeDocBlock
+{
+    /**
+     * @param $omittedTagType
```

Comment:
please remove this docblock, it's not needed

Reply:
It's very much needed; it's the entire point of the test. Currently the PhpDocExtractor throws a fatal error when getting the type from a PHPDoc tag like this one.

Reply:
I believe a comment explaining the point would be valuable here, to prevent somebody "fixing" it later :)